src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Input/Commands/Ordering/AbstractMoveCommand.cs

```diff
@@ -43,74 +44,19 @@ public AbstractMoveCommand(IPhysicalProjectTree projectTree, SVsServiceProvider
 
         protected override async Task<bool> TryHandleCommandAsync(IProjectTree node, bool focused, long commandExecuteOptions, IntPtr variantArgIn, IntPtr variantArgOut)
         {
-            var didMove = await TryMoveAsync(_configuredProject, node).ConfigureAwait(true);
+            var didMove = false;
+                
+            await _accessor.OpenProjectForWriteAsync(_configuredProject, project => didMove = TryMove(project, node)).ConfigureAwait(true);
```

Comment:
Why ConfigureAwait(true)?

Reply:
It shouldn't. since later I do a ConfigureAwait(true) for the publish before I call into Select.

Reply:
Expand on that?